.github/actions/pack_and_deploy/action.yml

```diff
@@ -17,7 +17,24 @@ runs:
             startsWith(github.ref, 'refs/heads/2.') ||
             startsWith(github.ref, 'refs/tags') ) }} ; then
           sudo apt-get -y update
-          sudo apt-get install -y procmail createrepo awscli reprepro
+          sudo apt-get install -y procmail awscli reprepro
+          # For RPM packages need to build 'createrepo_c' tool from
+          # sources till Github Actions will use ubuntu-21.04 where
+          # it be avaliable as DEB package for installation.
+          if [ ${OS} == 'el' -o ${OS} == 'fedora' -o ${OS} == 'opensuse-leap' ] ; then
+            git clone https://github.com/rpm-software-management/createrepo_c.git
```

Comment:
I would checkout a tag, `master` may change.

Reply:
Right, thought the same, but forgot to set, changed master to 0.17.1 (latest).